manifests/cluster-infra/4b_alertmanager_deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: alertmanager
+  labels:
+    app: alertmanager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: alertmanager
+  template:
+    metadata:
+      labels:
+        app: alertmanager
+    spec:
+      serviceAccountName: prometheus
+      containers:
+      - image: quay.io/prometheus/alertmanager:v0.17.0
+        args:
+        - "--web.external-url=http://{{ .DOMAIN_NAME }}/alertmanager"
+        - "--config.file=/etc/alertmanager/alertmanager.yml"
+        - "--storage.path=/alertmanager"
```

Comment:
If we're only running one AlertManager instance (which is probably fine for now), you can pass `--cluster.listen-address=` to [disable](https://github.com/prometheus/alertmanager#disabling-high-availability) the clustering (it will avoid any issue due to the cluster setup).